baseline-error-prone/src/test/java/com/palantir/baseline/errorprone/CollectionsIsEmptyTest.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * (c) Copyright 2023 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.baseline.errorprone;
+
+import com.google.errorprone.CompilationTestHelper;
+import java.util.List;
+import java.util.Set;
+import org.junit.jupiter.api.Test;
+
+public class CollectionsIsEmptyTest {
+    @Test
+    public void test_size_equals_zero() {
+        helper().addSourceLines(
+                        "Test.java",
+                        "import " + List.class.getCanonicalName() + ";",
+                        "class Test {",
+                        "  static boolean f(List<String> foo) {",
+                        "    // BUG: Diagnostic contains: foo.isEmpty()",
+                        "    return foo.size() == 0;",
+                        "  }",
+                        "}")
+                .doTest();
```

Comment:
Since we have an automated fix implemented, I'd use the refactoring helper and verify teh result instead of using the detection helper